eternalcore/src/test/java/com/eternalcode/core/configuration/language/LocaleComposerTest.java

```diff
@@ -0,0 +1,31 @@
+package com.eternalcode.core.configuration.language;
+
+import org.junit.jupiter.api.Test;
+import panda.std.Result;
+
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class LocaleComposerTest {
+
+    LocaleComposer composer = new LocaleComposer();
+
+    @Test
+    public void testDeserialize() {
+        String source = "en";
+        Result<Locale, Exception> result = composer.deserialize(source);
```

Comment:
```suggestion
        Result<Locale, Exception> result = this.composer.deserialize(source);
```
Z tego co widzę, to `composer` jest odwołaniem do fielda, więc wypadałoby oznaczyć to wyrazem `this` 😆